testing/src/Replay/WorkflowReplayer.php

```diff
@@ -33,7 +39,36 @@ public function __construct()
     }
 
     /**
-     * Replays workflow from a history that will be loaded from Temporal server.
+     * Replays a workflow from {@see History}
+     *
+     * @param non-empty-string|null $workflowType Workflow type name. If not specified, the workflow type will be
+     *        extracted from the first event in the history.
+     *
+     * @throws ReplayerException
+     */
+    public function replayHistory(
+        History $history,
+        ?string $workflowType = null,
```

Comment:
In most of the other SDKs we allow specifying the workflow id (since, oddly, that's not in history) rather than the type, which there isn't really much reason to specify since it ought to be extracted from the first event as you've done here.

If the user wants to try replaying a different wf type against that history, they simply can just register the workflow function with the same type name as is present in the history

Reply:
The workflow history already downloaded for specific workflow id via Run proto. We download history using id and then pass it to replay using this method - https://github.com/temporalio/roadrunner-temporal/blob/feature/replayer-api/rpc.go#L165

Reply:
Not sure how to pass id to that replayer.

Reply:
Oh, yeah, I guess Go doesn't even expose that right now. Don't worry about it then. I think we should remove wf type as a parameter though.

Reply:
That’s we can do since it’s already in history.